spec/cocina/models/mapping/descriptive/mods/abstract_spec.rb

```diff
@@ -98,54 +98,6 @@
     end
   end
 
-  describe 'Abstract with type "Summary"' do
-    it_behaves_like 'MODS cocina mapping' do
-      let(:mods) do
-        <<~XML
-          <abstract type="Summary">This is a summary.</abstract>
-        XML
-      end
-
-      let(:roundtrip_mods) do
-        <<~XML
-          <abstract type="summary">This is a summary.</abstract>
-        XML
-      end
-
-      let(:cocina) do
-        {
-          note: [
-            {
-              value: 'This is a summary.',
-              type: 'summary'
-            }
-          ]
-        }
-      end
-    end
-  end
-
-  describe 'Abstract with type "scope and content"' do
-    it_behaves_like 'MODS cocina mapping' do
-      let(:mods) do
-        <<~XML
-          <abstract type="scope and content">This is a scope and content note.</abstract>
-        XML
-      end
-
-      let(:cocina) do
-        {
-          note: [
-            {
-              value: 'This is a scope and content note.',
-              type: 'scope and content'
-            }
-          ]
-        }
-      end
-    end
-  end
-
```

Comment:
Specific type value does not matter, so this test is duplicative of the one at line 80.